packages/vite/src/node/build.ts

```diff
@@ -1101,24 +1101,28 @@ const getFileUrlFromFullPath = (path: string) =>
   `require('u' + 'rl').pathToFileURL(${path}).href`
 
```

Comment:
TODO: update the URL above after the rollup PR is merged
https://github.com/rollup/rollup/blob/0bcf0a672ac087ff2eb88fbba45ec62389a4f45f/src/ast/nodes/MetaProperty.ts#L145-L193